livechat/agent/web/client.py

```diff
@@ -49,15 +49,15 @@ def __init__(self, access_token: str, version: str, base_url: str):
         self.session = requests.Session()
         self.session.headers.update({'Authorization': access_token})
 
-    def modify_header(self, header: dict = None) -> None:
+    def modify_header(self, header: dict) -> None:
         ''' Modifies provided header in session object.
 
             Args:
                 header (dict): Header which needs to be modified.
         '''
         self.session.headers.update(header)
 
-    def remove_header(self, key: str = None) -> None:
+    def remove_header(self, key) -> None:
```

Comment:
hmmm so do we actually remove type hint here? Does this fix mypy error?

Reply:
it's not general rule, it helped in this case